src/components/NoticeComponent.tsx

```diff
@@ -195,7 +195,7 @@ class NoticeComponent extends React.Component<NoticeProps, NoticeState> {
                         id={"skipNoticeTimerText" + this.idSuffix}
                         key="skipNoticeTimerText"
                         className={this.state.countdownMode !== CountdownMode.Timer ? "hidden" : ""} >
-                            {this.state.countdownTime + "s"}
+                            {chrome.i18n.getMessage("NoticeTimeAfterSkip").replace("{seconds}", this.state.countdownTime)}
```

Comment:
We're using TS, so the `replaceValue` (second argument) to `.replace()` needs to be a string, but `countdownTime` is a number. Adding `.toString()` should make TS happy. Alternatively you can wrap the whole number value in `String()` - not sure which one is better here.
```suggestion
                            {chrome.i18n.getMessage("NoticeTimeAfterSkip").replace("{seconds}", this.state.countdownTime.toString())}
```

Reply:
Thank you! I pushed your changes just now